Resources/public/css/security.css

```diff
@@ -0,0 +1,60 @@
+.sonata-bc .tab-content {
+    padding-top: 3px;
+}
+
+.sonata-bc .row-fluid .role_table * {
+    margin-left: 0;
+}
+
+.sonata-bc .row-fluid .role_table [class*="span"].header, .role_table [class*="col"].header {
+    text-align: center;
+    padding-top: 5px;
+}
+
+.sonata-bc .row-fluid .role_table input {
+    width: auto;
+    text-align: center;
+}
+
+.role_table *[class*="col"], .role-editable, .role-editable *[class*="col"] {
+    padding: 0;
+    padding-left: 2px;
+}
+
+.sonata-bc .row-fluid .role_table .item, .sonata-bc .row-fluid .role_table .labelAdmin, .role_table .item, .role_table .labelAdmin {
+    padding-left: 17px;
+    padding-top: 5px;
+}
+
+.sonata-bc .row-fluid .role_table .labelPermission, .role_table .labelPermission {
```

Comment:
Inconsistant class naming. Please use kebab case for everything.
Also the role table is a top-level component, so it's class should be something like `sonata-admin-role-matrix`.